### ISSUE TITLE
Mobile Optimization: SEO

### ISSUE THREAD

**Issue by JonBrooksX** [jetlej/HE-6.0-Bugs#189]
I've just checked search console for mobile issues and it's saying there are 3 issues with our site design (	
http://www.highexistence.com/wp-content/themes/hethen-child/) on mobile. 

1) small font size.
2) touch elements too close. 
3) viewport not configured. 

Do you know of any easy ways we can solve these issues so that google views our site as more mobile-friendly?

**Comment by MSchirp** [comment]
Can you clarify the exact steps that can be done @JonBrooksX ? If these are more design-based, please confirm with @jetlej before initiating these changes.

**Comment by JonBrooksX** [comment]
Making these changes is out of my scope. This is for @dpatil-gmcs to look at, evaluate if it's correct (google is a bot after all) and make the changes as google suggests.

**Comment by dpatil-gmcs** [comment]
@JonBrooksX @MSchirp  Can you please let us know the site, which suggested the above recommendations. That way we can crosscheck after having it implemented.

**Comment by MSchirp** [comment]
@dpatil-gmcs This is related to #193 The sites can be found there.

**Comment by dpatil-gmcs** [comment]
@MSchirp  I had checked the following site for mobile reviews. https://developers.google.com/speed/pagespeed/insights/?url=highexistence.com&tab=mobile
and it says that the user experience is 100/100, Please provide the URL where you found the errors.

**Comment by MSchirp** [comment]
@dpatil-gmcs Yes, user experience is 100/100. We need to optimise for speed, which is 63/100. Please try to get this 100/100 too.

Same goes for blog posts, see: https://developers.google.com/speed/pagespeed/insights/?url=http%3A%2F%2Fhighexistence.com%2Fwhat-if-money-were-no-object-alan-watts-on-the-meaning-of-your-life%2F

@JonBrooksX Am I missing anything?

**Comment by JonBrooksX** [comment]
Image optimization is not a **top priority** according to Google. 

These are the two things we "should fix" according to Google:

**1) Eliminate render-blocking JavaScript and CSS in above-the-fold content**

Your page has 18 blocking script resources and 15 blocking CSS resources. This causes a delay in rendering your page.

None of the above-the-fold content on your page could be rendered without waiting for the following resources to load. Try to defer or asynchronously load blocking resources, or inline the critical portions of those resources directly in the HTML.

**2) Leverage browser caching**

Setting an expiry date or a maximum age in the HTTP headers for static resources instructs the browser to load previously downloaded resources from local disk rather than over the network.

**Comment by dpatil-gmcs** [comment]
@JonBrooksX  @MSchirp To eliminate the "render-blocking Javascript and CSS", We need to change the W3TC settings, The minify settings should be on.
But as per the @MSchirp 's talk with your server guy he has asked us to keep minify off in W3TC plugin.
Also, @jetlej  has asked not to change any W3TC setting without consulting him.
@jetlej @MSchirp  Can we on the minify settings in W3TC which I think would help us getting better score.

Please refer the following URLs for details:
1.  http://customizewp.com/eliminate-render-blocking-javascript-and-css-above-the-fold-content-in-wordpress-w3-total-cache/
2. http://www.digitalreborn.com/minify-wordpress-html-javascripts-css-w3-total-cache/

**Comment by MSchirp** [comment]
@dpatil-gmcs Please consult with Jordan and Hayden before making any changes.

**Comment by dpatil-gmcs** [comment]
@jetlej  @haydenjames  Can I on the minify in W3TC plugin settings. Just to make the page loading faster. Please refer the above URLs.

**Comment by dpatil-gmcs** [comment]
@MSchirp  WE have enabled the minify settings, on the TEST site, as per hayden there were some problems occoured when minification was on, So I think let us first enable it on TEST site and observe it for few days, So that we can be sure about the problem if any exists, and then plan to push it to LIVE site.

Please let us know your thoughts on it!

**Comment by dpatil-gmcs** [comment]
@jetlej @haydenjames, As we have ebabled the minify settings on TEST site, can you please verify or provide us with the steps to check if the problem which @haydenjames discussed few days back persists.
So that we can go LIVE and get the better Google Insights score.

**Comment by haydenjames** [comment]
@dpatil-gmcs I'm not sure what the bug was. Jordan will better be able to provide steps. The best thing to do it test during off hours or during the lowest traffic day. This way, if the issue persists you can revert without to much disrupt to end-users. 

If the TEST site is a recent clone/full copy of the live's site's Wordpress directory and database, then sounds like you should be ok.

**Comment by dpatil-gmcs** [comment]
@haydenjames @MSchirp  Ok then, I will ask Jordan about the bug, and try to reproduce it on the test site, If the bug is not reproducible or seems to be fixed, then I think we are good to enable the minify settings on the LIVE site.
Let us know your thoughts on it!

**Comment by dpatil-gmcs** [comment]
@jetlej  It seems that, enabling the minify settings make some problem with the User profile picture/avatar. Can you please let us know the problem that you had faced when you enabled minify settings in W3TC settings?
Also can you please let us know the steps to reproduce the problem?

**Comment by haydenjames** [comment]
If I remember correctly. The fix for the avatar issue was to add the
related .js or .CSS file to the exclude list. I will search through email
from 2 years ago when I believe Jordan may have provided me with the name
of the file that when minified would cause the avatar bug.

Will follow up if I'm able to find the file name.

Hayden

**Comment by dpatil-gmcs** [comment]
@haydenjames @MSchirp   I have communicated Jordan about it, He says he doesn't remember the exact problem but Minify always created problems, So he suggest to keep it off.

**Comment by MSchirp** [comment]
Is minify the only way to implement google's suggested changes?
